src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -784,7 +784,10 @@ def build_vmss_resource(cmd, name, naming_prefix, location, tags, overprovision,
         vmss_properties['platformFaultDomainCount'] = platform_fault_domain_count
 
     if ultra_ssd_enabled is not None:
-        vmss_properties['virtualMachineProfile']['additionalCapabilities'] = {'ultraSSDEnabled': ultra_ssd_enabled}
+        if cmd.supported_api_version(min_api='2019-03-01', operation_group='virtual_machine_scale_sets'):
```

Comment:
I'm assuming these changes came from @adewaleo directly into the release branch, and that's why this is the first time they're being applied to the `dev` branch. Is that correct?

Reply:
Nevermind, I see that in the commit log above. These changes came from #9421.